dcos/util.py

```diff
@@ -548,3 +548,41 @@ def io_exception(path, errno):
 
 
 logger = get_logger(__name__)
+
+
+def _get_http_proxy_url():
+    """ Returns http proxy server url from config
+
+    :returns: Http Proxy server URL
+    :rtype: str
+    """
+    config = get_config()
+    return config.get('core.http_proxy')
```

Comment:
I am not sure that we want to use this pattern. Normally we read and parse the config file once and pass the option or parts of the object down to the functions that need it. I think we should keep that pattern.